aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md

```diff
@@ -46,53 +48,119 @@ If the target framework is .NET Core, the **OS** drop-down allows for the select
+* *.dockerignore* &mdash; Lists the file and directory patterns to exclude when generating a build context.
+* *docker-compose.yml* &mdash; The base [Docker Compose](https://docs.docker.com/compose/overview/) file used to define the collection of images to be built and run with `docker-compose build` and `docker-compose run`, respectively.
+* *docker-compose.override.yml* &mdash; An optional file, read by Docker Compose, with configuration overrides for services. Visual Studio executes `docker-compose -f "docker-compose.yml" -f "docker-compose.override.yml"` to merge these files.
+
+The *docker-compose.yml* file references the name of the image that's created when the project runs:
+
+[!code-yaml[](visual-studio-tools-for-docker/samples/2.0/docker-compose.yml?highlight=5)]
 
 In the preceding example, `image: hellodockertools` generates the image `hellodockertools:dev` when the app runs in **Debug** mode. The `hellodockertools:latest` image is generated when the app runs in **Release** mode.
 
-Prefix the image name with the [Docker Hub](https://hub.docker.com/) username (for example, `dockerhubusername/hellodockertools`) if the image will be pushed to the registry. Alternatively, change the image name to include the private registry URL (for example, `privateregistry.domain.com/hellodockertools`) depending on the configuration.
+Prefix the image name with the [Docker Hub](https://hub.docker.com/) username (for example, `dockerhubusername/hellodockertools`) if the image is to be pushed to the registry. Alternatively, change the image name to include the private registry URL (for example, `privateregistry.domain.com/hellodockertools`) depending on the configuration.
+
+### Service Fabric
+
+In addition to the base [Prerequisites](#prerequisites), the [Service Fabric](/azure/service-fabric/) orchestration solution demands the following prerequisites:
+
+* [Microsoft Azure Service Fabric SDK](https://www.microsoft.com/web/handlers/webpi.ashx?command=getinstallerredirect&appid=MicrosoftAzure-ServiceFabric-CoreSDK) version 2.6 or later
+* Visual Studio 2017's **Azure Development** workload
+
+Service Fabric doesn't support running Linux containers in the local development cluster on Windows. If the project is already using a Linux container, Visual Studio prompts to switch to Windows containers.
+
+The Visual Studio Tools for Docker do the following tasks:
+
+* Adds a *&lt;project_name&gt;Application* **Service Fabric Application** project to the solution.
+* Adds a *Dockerfile* and a *.dockerignore* file to the ASP.NET Core project. If a *Dockerfile* already exists in the ASP.NET Core project, it's renamed to *Dockerfile.original*. A new *Dockerfile*, similar to the following, is created:
+
+    [!code-dockerfile[](visual-studio-tools-for-docker/samples/2.1/HelloDockerTools/Dockerfile)]
+
+* Adds an `<IsServiceFabricServiceProject>` element to the ASP.NET Core project's *.csproj* file:
```

Comment:
I think we're bolding these bad boys ... elements in either markup or project config files. I guessed a little bit from our markup guidelines anyway. Let me know if I should stop doing it.

> an \*\*\&lt;IsServiceFabricServiceProject\&gt;\*\* element

Reply:
I reserve bolding for UI elements. I treat XML elements like code.